src/nnfusion/core/kernels/cuda_gpu/cuda_emitter.hpp

```diff
@@ -189,18 +189,44 @@ namespace nnfusion
                         auto ir = nnfusion::op::get_translation(ctx->gnode);
                         if (!ir.empty())
                         {
-                            auto info = m_antares_ke_imp->autogen(ir);
-                            antares_code = info.first;
-                            m_is_tuned = info.second;
+                            const char annotation[] = "## @annotation: ";
+                            int pos = ir.find(annotation);
```

Comment:
Need to merge similar logic with codes from cpu_emitter

Reply:
May i ask why?

Reply:
We may have different emitter for many backends, and we will also change the annotation format to make it stronger. So how to ensure every format-parsing version is up-to-date?

Reply:
just updated.